src/api/Configuration.php

```diff
@@ -26,7 +26,7 @@ class Configuration extends AbstractConfiguration
      */
     public function isEnabled()
     {
-        return \ddtrace_config_trace_enabled();
+        return $this->boolValue('trace.enabled', true);
```

Comment:
Did we mean to revert these changes?

Reply:
yep, we don't want to use new functions at all in `/api`